src/components/Article.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useParams } from 'react-router-dom';
+import CommentsList from './CommentsList'
+import Vote from './Vote';
+
+const Article = () => {
+    const { article_id } = useParams();
+    const [article, setArticle] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
+        axios.get(`https://sorei9240-nc-news.onrender.com/api/articles/${article_id}`)
+            .then((response) => {
+                setArticle(response.data.article);
+                setIsLoading(false);
+            })
```

Comment:
Would be better if all the api calls are abstracted into another file to keep the components neat